.circleci/config.yml

```diff
@@ -14,7 +14,14 @@ commands:
       - run:
           name: Install gotestsum, kind, kubectl, and helm
           command: |
-            go get gotest.tools/gotestsum
+            wget https://golang.org/dl/go1.16.5.linux-amd64.tar.gz
+            sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.16.5.linux-amd64.tar.gz
+            rm go1.16.5.linux-amd64.tar.gz
+            echo 'export PATH=$PATH:/usr/local/go/bin' >> $BASH_ENV
+
+            wget https://github.com/gotestyourself/gotestsum/releases/download/v1.6.4/gotestsum_1.6.4_linux_amd64.tar.gz
+            sudo tar -C /usr/local/bin -xzf gotestsum_1.6.4_linux_amd64.tar.gz
+            rm gotestsum_1.6.4_linux_amd64.tar.gz
```

Comment:
`go get` will install dependencies too, and that was creating conflicts when we tried to to restore cache (e.g. https://app.circleci.com/pipelines/github/hashicorp/consul-helm/3267/workflows/38b0913e-dde9-4244-aeae-7a4752fa75cc/jobs/14103)

Reply:
this is an excellent find!!